src/specs/api.v2.yaml

```diff
@@ -969,6 +1061,8 @@ components:
       $ref: './models/experiment-bodies/ExperimentInfo.v2.yaml'
     PatchExperiment:
       $ref: './models/experiment-bodies/PatchExperiment.v2.yaml'
+    GetExperimentUsers:
+      $ref: './models/access-bodies/ExperimentUsers.v2.yaml'
```

Comment:
Call th file `GetExperimentUsers`?

Reply:
This has been remove in a previous change